site/styles/_variables.scss

```diff
@@ -108,7 +109,7 @@ $carousel-bg-color: $body-bg !default;
 $carousel-height: 470px !default;
 $carousel-height-sm: 470px !default;
 $carousel-height-md: 620px !default;
-$carousel-caption-top: 150px !default;
+$carousel-caption-top: 100px !default;
```

Comment:
Does the caption need to be higher on mobile?

Reply:
Not sure how that got there. I've changed it back.